test/app.test.js

```diff
@@ -30,7 +30,7 @@ describe('app', function() {
       return redirects.indexOf(route) === -1;
     }).map(function(url) {
       if (url === '/') return '/';
-      return '/' + url;
+      return '/' + url + '/';
```

Comment:
303 is when it's a "moved" for things that are not plain GET operations - it's a "moved and what you expected to happen probably didn't happen" code